### ISSUE TITLE
Proprietary fields

### ISSUE THREAD

**Issue by Spirtech-GT4** [lbureau-billettiqueservices/GT4_support_schema#10]
To avoid conflicts between proprietary fields and any future evolution of the standard, there should be a naming rule for proprietary fields.
For example starting with an underscore, e.g. "_my_data".

**Comment by lbureau-billettiqueservices** [comment]
On one hand it's a good idea, on the other i'm not sure if the restriction is really important.

First, non-declared fields shouldn't be used by tools by default. If it's not part of the schema, it should be ignored.

Second, if a new field is added by a tool, and we realize it's an interesting field to add and normalize, this would require the original tool maker to update its field name to adhere to the standard again. I feel like this limits potential contributions a bit too much.

**Comment by Spirtech-GT4** [comment]
Indeed, both options having pro & cons, the GT4 will have to chose.

**Comment by lbureau-billettiqueservices** [comment]
"free_data" structure with prefixed fields required for proprietary fields to avoid collisions.

**Comment by lbureau-billettiqueservices** [comment]
Need authority on prefixes integrated in the format.
